native/python3/Makefile

```diff
@@ -29,10 +29,11 @@ PIP_NATIVE = $(WORK_DIR)/../../../native/$(PKG_NAME)/work-native/install/usr/loc
 
 .PHONY: nativepython3_post_install
 nativepython3_post_install: $(WORK_DIR)/python-native.mk
-	@$(MSG) Installing setuptools, pip and cffi
-	@$(RUN) wget https://bootstrap.pypa.io/ez_setup.py -O - | $(PYTHON)
+	@$(MSG) Installing pip, setuptools, cffi and cross env
 	@$(RUN) wget https://bootstrap.pypa.io/get-pip.py -O - | $(PYTHON)
+	@$(PIP) install "setuptools==44.0.0"
```

Comment:
I aligned it with the version under `cross/setuptools` instead of using `ez_setup` which is now deprecated